### ISSUE TITLE
[Question][Hw1] Adding custom testcase

### ISSUE THREAD

**Issue by uk-jeong** [prosyslab-classroom/cs492-program-reasoning#145]
Name: JeongUk Lee

I created several custom test cases (example_customX.c/.input/.expected) in order to increase my code coverage.
However, these test cases are not being included in the "make coverage" command.
When I include these test cases in the test/dune file, the coverage points increase.
Should I modify the test/dune file, or am I missing something?

**Comment by goodtaeeun** [comment]
Yes, you need to modify the `test/dune` file accordingly to the new test cases.
